### PR TITLE
Use icons from dreamstore icon pack

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What is the purpose of this pull request?
Use icons from dreamstore icon pack

#### How should this be manually tested?
[Workspace](https://lucas--storecomponents.myvtex.com/eletronicos/d?map=c%2Cc&rest=Smartphones).

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [x] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by brunojdo** [comment]
@matheusps change to your approach.

**Comment by matheusps** [comment]
> @matheusps change to your approach.

Done ✅. You can access through this [workspace](https://matheusps--storecomponents.myvtex.com/iPho/s?map=ft)

File: react/images/Arrow.js [hunk 1]
```diff
@@ -1,42 +1,41 @@
...
+      xmlnsXlink="http://www.w3.org/1999/xlink"
+      width={width}
+      height={height}
+      viewBox="0 0 14 10"
+      fill="none"
+    >
+      <Use id={iconId} />
+    </svg>
+  )
+}
```
Comment (vitoriaHeliane) [inline react/images/Arrow.js]: Add a white line here.

File: react/images/InlineGrid.js [hunk 2]
```diff
@@ -1,20 +1,21 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import Use from 'vtex.react-use-svg/Use'
 
 export default function InlineGrid({ active }) {
   const color = active ? '#828282' : '#ececec'
```
Comment (vitoriaHeliane) [inline react/images/InlineGrid.js]: You can use Tachyons here if add a `div` parent and set its color by using tachyons' classes.

File: react/images/Grid.js [hunk 3]
```diff
@@ -1,19 +1,21 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import Use from 'vtex.react-use-svg/Use'
 
 export default function Grid({ active }) {
   const color = active ? '#828282' : '#ececec'
```
Comment (vitoriaHeliane) [inline react/images/Grid.js]: You can use Tachyons here if add a `div` parent and set its color by using tachyons' classes.

File: react/images/Arrow.js (hunk 1 above)
Comment (vitoriaHeliane) [inline react/images/Arrow.js]: Add a white line here.

File: react/images/Grid.js (hunk 3 above)
Comment (vitoriaHeliane) [inline react/images/Grid.js]: You can use Tachyons here if add a `div` parent and set its color by using tachyons' classes.

File: react/images/Arrow.js (hunk 1 above)
Comment (vitoriaHeliane) [inline react/images/Arrow.js]: Add a white line here.

File: react/images/Grid.js (hunk 3 above)
Comment (vitoriaHeliane) [inline react/images/Grid.js]: You can use Tachyons here if add a `div` parent and set its color by using tachyons' classes.

File: manifest.json [hunk 4]
```diff
@@ -28,6 +28,7 @@
     "vtex.styleguide": "6.x",
     "vtex.product-summary": "1.x",
     "vtex.store-graphql": "2.x",
-    "vtex.store": "1.x"
+    "vtex.store": "1.x",
+    "vtex.react-use-svg": "1.x"
```
Comment (brunojdo) [inline manifest.json]: 0.x

File: react/images/Check.js [hunk 5]
```diff
@@ -1,26 +1,18 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import Use from 'vtex.react-use-svg/Use'
 
 export default function Check({ size }) {
   return (
     <svg
+      xmlns="http://www.w3.org/2000/svg"
```
Comment (brunojdo) [inline react/images/Check.js]: This is always the same, why you don't put in Use?

Comment (matheusps) [reply react/images/Check.js]: I think puting all of this in a single component would be a little strange. An idea that came in mind is to create another component called Svg, that wraps svg tags with default values for props xmlns and xmlnsXlink. Something like:

```
...
const Svg = ({ children, ...props}) => (
  <svg {...props}>
    { children }
  </svg>
);

Svg.defaultProps = {
  xmlns: 'http://www.w3.org/2000/svg',
  xmlnsXlink='http://www.w3.org/1999/xlink'
};
```

Using the component would be like: 
```
...
<Svg 
  width={size}
  height={size}
  viewBox="0 0 9 9"
  fill="none"
>
  <Use id="check" />
</Svg>
```

This Component would be useful in any other place that we use svgs. What do you think ?

File: manifest.json (hunk 4 above)
Comment (matheusps) [reply manifest.json]: Fixed

File: react/images/Arrow.js (hunk 1 above)
Comment (matheusps) [reply react/images/Arrow.js]: Fixed

File: react/images/Grid.js (hunk 3 above)
Comment (matheusps) [reply react/images/Grid.js]: Done

File: react/images/CheckTick.js [hunk 6]
```diff
@@ -1,21 +1,20 @@
...
-      </svg>
-    )
-  }
+
+const CheckTick = () => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      xmlnsXlink="http://www.w3.org/1999/xlink"
+      width="11"
```
Comment (vitoriaHeliane) [inline react/images/CheckTick.js]: I think that the icons size should be able to change by props.

File: react/images/GridIcon.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+GridIcon.propTypes = {
+  /** Whether the icon is active */
+  active: bool,
+  activeClassName: string,
+  muttedClassName: string,
+  width: string,
+  height: string,
+  fill: string,
+  iconID: string.isRequired
```
Comment (lucasecdb) [inline react/images/GridIcon.js]: rename to `icon` or `type`

**Pull request closed without merging**